### ISSUE TITLE
A more precise "blocks"/"modular" content type

### ISSUE THREAD

**Issue by grantholle** [statamic/ideas#812]
I'm new to Statamic so I may be missing something.

I currently have a Bard field in my Pages blueprint that holds all of my content. I've organized different content type fieldsets and in general, it gets the job done. My biggest roadblock is when it needs to be localized. I've set up the fieldsets with the right localizable fields, but inside of a Bard field it isn't possible to be so precise, because if I change one small detail of a fleldset/block in the Bard, the entire Bard field is now out of sync. Any change I make to the default site doesn't get reflected in the other site because the Bard field is technically different.

Basically, I need a way to add arbitrary blocks as my content where, if they're not localized, still sync with the default/origin site. Like, the functionality of a Bard.... without the Bard wrapper. If that makes sense.
